site/tsconfig.json

```diff
@@ -1,14 +1,14 @@
 {
   "compilerOptions": {
     "baseUrl": ".",
-    "module": "esnext",
+    "module": "nodenext",
```

Comment:
any reason this was needed?

Reply:
yea the snippets files weren't correctly resolving the `/react` and `/config` imports so I did this to match what we have in other tsconfigs